sdk/analytics/src/main/java/com/klaviyo/analytics/Klaviyo.kt

```diff
@@ -82,9 +84,27 @@ object Klaviyo {
             resetProfile()
         }
 
-        UserInfo.externalId = profile.externalId ?: ""
-        UserInfo.email = profile.email ?: ""
-        UserInfo.phoneNumber = profile.phoneNumber ?: ""
+        if (profile.email != null &&
```

Comment:
I think there might be a cleaner place for this to live, in `ProfileOperationQueue`

Reply:
Then we'd only have to check idenfitiers in one place, no code changes need in `Klaviyo.kt`